build_tools/docker/riscv/Dockerfile

```
@@ -8,11 +8,11 @@
 
 FROM gcr.io/iree-oss/base@sha256:5d43683c6b50aebe1fca6c85f2012f3b0fa153bf4dd268e8767b619b1891423a AS install-riscv
 WORKDIR /install-riscv
-RUN wget "https://storage.googleapis.com/iree-shared-files/toolchain_iree_rvv-intrinsic.tar.gz"
-RUN tar -xf "toolchain_iree_rvv-intrinsic.tar.gz" -C /usr/src/
-RUN wget "https://storage.googleapis.com/iree-shared-files/toolchain_iree_rv32.tar.gz"
-RUN tar -xf "toolchain_iree_rv32.tar.gz" -C /usr/src/
-RUN wget "https://storage.googleapis.com/iree-shared-files/qemu-riscv.tar.gz"
+RUN wget -nv "https://storage.googleapis.com/iree-shared-files/toolchain_iree_20220918.tar.gz"
+RUN tar -xf "toolchain_iree_20220918.tar.gz" -C /usr/src/
+RUN wget -nv "https://storage.googleapis.com/iree-shared-files/toolchain_iree_rv32_20220918.tar.gz"
+RUN tar -xf "toolchain_iree_rv32_20220918.tar.gz" -C /usr/src/
+RUN wget -nv "https://storage.googleapis.com/iree-shared-files/qemu-riscv.tar.gz"
```

Can you use long options here? Short options are great for easily typing things on the command line, but are not very readable when checked in to source control. I try to remember to use long options in all our scripts, though notably we haven't done that for tar consistently. `-v` is usually pretty well known, but I had to look up `-nv` (it's also pretty hard to search short options, and having them be multiple characters is not typical).

```suggestion
RUN wget --no-verbose "https://storage.googleapis.com/iree-shared-files/toolchain_iree_20220918.tar.gz"
RUN tar -xf "toolchain_iree_20220918.tar.gz" -C /usr/src/
RUN wget --no-verbose "https://storage.googleapis.com/iree-shared-files/toolchain_iree_rv32_20220918.tar.gz"
RUN tar -xf "toolchain_iree_rv32_20220918.tar.gz" -C /usr/src/
RUN wget --no-verbose "https://storage.googleapis.com/iree-shared-files/qemu-riscv.tar.gz"
```

---

Done. :) Thanks for the comment.